README.md

```diff
@@ -11,79 +11,71 @@ description: "Welcome to the Knowledge Mining Solution Accelerator!"
+> Note: This guide uses the AI enrichment feature of Cognitive Search. AI enrichment allows you to ingest many kinds of data (documents, text files, images, scanned docs, and more), extract their contents, enrich and transform it, and then index it for exploration purposes. To learn more about this feature, see the [AI in Cognitive Search](https://docs.microsoft.com/azure/search/cognitive-search-concept-intro) doc.
 
-Once finished, you'll have a web app ready to search against your data like in the image below:
-![screenshot](images/ui.PNG)
+Once you're finished, you'll have a web app ready to search your data.
+
+![A web app showing several resources and their lists of searchable tags](images/ui.PNG)
 
 ## Prerequisites
-In order to successfully complete your solution, you will need to have access to and or provisioned the following:
-1. Access to an Azure subscription
-2. Visual Studio 2017 or newer
-3. [Postman](https://www.getpostman.com/) for making API calls
-4. Documents uploaded to any data source [supported by Azure Search Indexers](https://docs.microsoft.com/en-us/azure/search/search-indexer-overview).  This solution accelerator uses Azure Blob Storage as a container for source data files. Sample documents are provided in the *sample_documents* folder.
 
-## Training
-The directions provided for this repository assume fundemental working knowledge of Azure Portal, Azure Functions, Azure Search, Visual Studio and Postman.  
+In order to successfully complete your solution, you'll need to gain access and provision the following resources:
 
-For additional training and support, please see:
- 1. [Knowledge Mining Bootcamp](https://github.com/Azure/LearnAI-KnowledgeMiningBootcamp)
- 2. [Full Cognitive Search Documentation](https://docs.microsoft.com/en-us/azure/search/cognitive-search-resources-documentation)
+* Azure subscription - [Create one for free](https://azure.microsoft.com/free/)
+* [Visual Studio 2017 or later](https://visualstudio.microsoft.com/downloads/)
+* [Postman](https://www.getpostman.com/) for making API calls
+* Documents uploaded to any data source supported by Azure Search Indexers. For a list of these, see [Indexers in Azure Cognitive Search](https://docs.microsoft.com/azure/search/search-indexer-overview). This solution accelerator uses Azure Blob Storage as a container for source data files. You can find sample documents in the **sample_documents/** folder.
 
-## Getting Started and Process Overview
-Clone/download this repo onto your computer and then walk through each of these folders in order, following the steps outlined in each of the README files.  After completion of all steps, you will have a working end-to-end solution with the following architecture:
+The directions provided in this guide assume you have a fundamental working knowledge of the Azure portal, Azure Functions, Azure Search, Visual Studio and Postman. For additional training and support, please see:

```

Comment:
Let's say Azure Cognitive Search (not just Azure Search -- as the name changed recently at Ignite)